website/blog/2021/06/28/why-we-need-Apache-APISIX.md

```diff
@@ -25,7 +25,7 @@ Hello everyone, I'm happy to share a topic that I'm excited about, "Why do you n
 The reason why we are doing a replacement project for NGINX and Kong is actually related to the background of our backend architecture evolution, and I will start by sharing with you the backend architecture evolution process, which is very important.
 
 I'll start by sharing with you the evolution of the backend architecture, which is very important.
-![membphis](https://static.apiseven.com/202108/1646299637523-b074acde-e902-483b-bc27-7ebe75e675cb.jpg)
+![membphis](https://static.apiseven.com/202108/1646320281711-c7e446ce-91f3-4494-93de-8c4b9f7936a2.jpg)
```

Comment:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/6814606/156678222-59df3de0-9d17-415c-851a-42f8bfcaa6dd.png">

Reply:
> <img alt="image" width="377" src="https://user-images.githubusercontent.com/6814606/156678222-59df3de0-9d17-415c-851a-42f8bfcaa6dd.png">

It means change to "Apache apifix PMC"?

Reply:
It should be `Apache APISIX PMC` or `Apache APISIX PMC Member`

Reply:
> It should be `Apache APISIX PMC` or `Apache APISIX PMC Member`

get